modules/ipi-preparing-reinstall-cluster-bare-metal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
+
+[id="ipi-preparing-reinstall-cluster-bare-metal_{context}"]
+
+= Preparing to reinstall a cluster on bare-metal
+Before you reinstall a cluster on bare-metal, you must perform cleanup operations.
+
+.Procedure
+. Remove or reformat the disks for the bootstrap, master, and worker nodes. If you are working in a hypervisor environment, you will need to re-add any disks you removed. You are now starting with clean disks for the bootstrap, master, and worker nodes.
```

Comment:
```suggestion
. Remove or reformat the disks for the bootstrap, <control-plane>, and worker nodes. If you are working in a hypervisor environment, you must add any disks you removed. You are now starting with clean disks for the bootstrap, <control-plane>, and worker nodes.
```

- I think that the last sentence can be cut. It doesn't add anything, and it is one more thing to wrangle with the conscious language issue mentioned in the comment above. 
- The term _re-add_ is not in the Merriam-Webster dictionary. I could not find guidance on the term in the SSG or ISG.
- [ISG](https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense) advises against using future tense.

Reply:
Done.